dev/src/user.clj

```diff
@@ -2,7 +2,6 @@
   (:require
    [hawk.assert-exprs]
    [metabase.bootstrap]
-   metabase.query-processor-test.test-mlv2
```

Comment:
Discussed here https://metaboat.slack.com/archives/C010L1Z4F9S/p1681319052854699?thread_ts=1681241917.937639&cid=C010L1Z4F9S -- loading this as part of `user.clj` is a little iffy since it loads up a huge chunk of the system as a side-effect. Fixing this has been on my TODO list since last week. It's better to only load this when we actually need it, i.e. in `metabase.query-processor`.